src/mpid/ch4/netmod/ofi/ofi_types.h

```diff
@@ -388,6 +336,7 @@ typedef struct {
     /* Window/RMA Globals */
     void *win_map;
     uint64_t rma_issued_cntr;
+    void *win_id_allocator;
```

Comment:
I feel like this should now be called `mr_key_allocator`.

Reply:
Also, if I understood correctly, this single allocator is now used for 
- generating window ID for RMA (=equal to MR key for RMA)
- generating MR key for pt2pt LMT

These design choices have to be written somewhere, probably here.